openpype/hosts/maya/plugins/publish/extract_playblast.py

```diff
@@ -138,8 +138,10 @@ def process(self, instance):
         self.log.debug("playblast path  {}".format(path))
 
         collected_files = os.listdir(stagingdir)
+        pattern_frame = [r'\.(?P<index>(?P<padding>0*)\d+)\.\D+\d?$']
         collections, remainder = clique.assemble(collected_files,
-                                                 minimum_items=1)
+                                                 minimum_items=1,
+                                                 patterns=pattern_frame)
```

Comment:
Seems unrelated to the PR description.

Reply:
my bad, this is related to another ticket.

Reply:
> Seems unrelated to the PR description.

coming from #3683 ?

Reply:
yes, it is